src/sidebar/templates/top-bar.html

```diff
@@ -63,7 +63,7 @@
        ng-click="vm.onShowHelpPanel()"
        title="Help"
        aria-label="Help">
-       <svg-icon name="'help'"></svg-icon>
+       <svg-icon name="'help'" class-name="'svg-icon__help'"></svg-icon>
```

Comment:
The "Help" icon is part of the `top-bar` component rather than the `svg-icon` component, so the class name should reflect that (ie. `top-bar__help-icon`)